src/assetindex/assetIndex.es6

```diff
@@ -155,10 +155,13 @@ const initTable = () => {
         }
 
         function marketsDropdown(market_submarkets) {
+            const markets_array = Object.keys(market_submarkets);
+            const markets_reversed = markets_array.length > 1 ? Object.assign([], markets_array, {[2]: markets_array[3], [3]: markets_array[2]}) : markets_array; // handle norway clients (handle translation)
```

Comment:
I think it's possible to add a function for `markets_reversed` and add it to marketUtils?